src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -102,6 +106,22 @@ public CommandResult executeUndoableCommand() throws CommandException {
     }
 
     /**
+     * Does the appropriate list status checks on {@code personToEdit}
+     * and changes the list status of {@code editedPerson} accordingly.
+     * Returns {@code editedPerson}.
```

Comment:
should use javadoc annotations for return: `@return`

Reply:
Noted.